Add link to migrating to SaaS workshop (#9746)
getsentry/sentry-docs: docs/product/accounts/choose-your-data-center/index.mdx

### Self-hosted

If you have a self-hosted Sentry account, you can [follow these instructions](product/accounts/migration/) to migrate your data.

<Alert>

> If you're interested in learning more about switching to Sentry SaaS, [Sign up here](https://sentry.io/resources/migrate-to-sentry-saas-workshop/) to join a live workshop on April 30th. You'll learn about our relocation tooling and get a chance to ask questions live.

Reviewer: ```suggestion
If you're interested in learning more about switching to Sentry SaaS, [sign up here](https://sentry.io/resources/migrate-to-sentry-saas-workshop/) to join a live workshop on April 30th. You'll learn about our relocation tooling and get a chance to ask questions live.
```

> <Alert>
> If you're interested in learning more about switching to Sentry SaaS, [Sign up here](https://sentry.io/resources/migrate-to-sentry-saas-workshop/) to join a live workshop on April 30th where you can learn about our relocation tooling and support, and get your questions answered in real time.
> </Alert>

Author: ```suggestion
<Alert>
If you're interested in learning more about switching to Sentry SaaS, [Sign up here](https://sentry.io/resources/migrate-to-sentry-saas-workshop/) to join a live workshop on April 30th. You'll learn about our relocation tooling and get a chance to ask questions live.
</Alert>
```
